src/main/java/org/commonjava/util/sidecar/services/ProxyService.java

```diff
@@ -152,15 +152,25 @@ long readTimeout()
         byte[] bytes = IOUtils.toByteArray( is );
         Buffer buf = Buffer.buffer( bytes );
 
-        if ( getBuildConfigId() != null && path.split( "/" )[2].equals( "maven" ) )
+        return normalizePathAnd( path, p -> classifier.classifyAnd( p, request, ( client, service ) -> wrapAsyncCall(
```

Comment:
Does this mean folo tracking not needed in POST?

Reply:
Yes, every upload uses PUT method, POST is only for promotion